filter.lua

```diff
@@ -74,10 +72,19 @@ local reply = {
 
 local action = xdp.action
 
+local action_name = (function ()
+        local map = {}
+        for name, action in pairs(action) do
+                map[action] = name
+        end
+        return map
+end)()
```

Comment:
not that I'm against it.. just found this pattern a bit weird.. usually we do that in Lua..
```suggestion
local action_name = {}
for name, action in pairs(action) do
   action_name[action] = name
end
```